connector_magento/components/backend_adapter.py

```diff
@@ -101,27 +47,84 @@ def location(self):
         return location
 
 
```

Comment:
It seems to be a bug. `use_auth_basic` is always false even if checked on the backend.
I try to determine while but it is not clear for me.